src/index.html

```diff
@@ -7,6 +7,331 @@
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">APPLE</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">SAMSUNG</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">SMARTPHONES</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">GADGETS</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">TABLETS</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">PHOTO</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">VIDEO</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div class="card card--position" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+          class="card__image"
+        >
+
+        <div class="card__info">
+          <h1 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__rating rating">
+            <div class="rating__stars stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="rating__reviews">Reviews: 5</div>
+          </div>
+          <div class="card__price">
+            <span class="card__price-text">Price:</span> <span class="card__price-sum">$2,199</span>
+          </div>
+        </div>
+
+        <a
+          href="#"
+          class="card__link"
+          data-qa="card-hover"
+        >
+          BUY
+        </a>
+      </div>
+      <div class="card card--position">
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+          class="card__image"
+        >
+
+        <div class="card__info">
+          <h1 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__rating rating">
+            <div class="rating__stars stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="rating__reviews">Reviews: 5</div>
```

Comment:
Use semantic tags for text content like `<p>, <h2>, <span>` and so on


fix everywhere